FAQ Section Updates

anandani4136 opened the pull request.

aniketgargya committed:
Add files via upload

aniketgargya committed:
Merge pull request #40 from HackIllinois/aniketgargya-patch-1

🍱 update sponsorship document

ashayp22 committed:
Updating HackIllinois 2023 Sponsorship PDF (#42)

ashayp22 committed:
Update Sponsorship Document Metadata Title (#44)

anandani4136 committed:
:sparkles: Starting registration work

This reverts commit ecb6dfa13b264555b8adcc9d454c52a53935a0b8.

anandani4136 committed:
:bug: Fixed static height :sparkles: added MoonLight

anandani4136 committed:
:lipstick: Whack a Mole, navigation testing

aniketgargya committed:
Merge pull request #45 from HackIllinois/nextjs

add 2023 Hero to homepage

aniketgargya committed:
:truck: add /sponsor reroute

aniketgargya committed:
:truck: add /sponsor reroute (#46)

aniketgargya committed:
:passport_control: possibly re-add auth route

aniketgargya committed:
Merge pull request #47 from HackIllinois/auth

Re-add /auth for mobile authentication

aniketgargya committed:
:beers: add the forgotten auth.tsx page

aniketgargya committed:
Merge pull request #48 from HackIllinois/auth

:beers: add the forgotten auth.tsx page

aniketgargya committed:
:bug: use router.isReady before processing params in /auth

aniketgargya committed:
Merge pull request #49 from HackIllinois/auth

:bug: use router.isReady before processing params in /auth

anandani4136 committed:
:lipstick: Mole spring animation part 1

anandani4136 committed:
:lipstick: mole transition refined

anandani4136 committed:
:sparkles: Added FAQs :balloon:

anandani4136 committed:
:lipstick: Show Hero

anandani4136 committed:
:art: Removed Welcome Screen, Simplified Mole Code

aniketgargya commented:
Looks really good!
Just make a couple very small changes:
- Increase the font weight of the FAQ text (to 400 or 500)
- Have the links on the during and after section also be white
- Use `cursor: pointer;` when hovering over the clickable balloons to make sure people can see them (if possible).

Besides that, great job!

aniketgargya committed:
:children_crossing: add temporary background blue color to homepage

aniketgargya committed:
Merge pull request #52 from HackIllinois/temp-changes

:children_crossing: add temporary background blue color to homepage

anandani4136 committed:
:lipstick: Implemented recommended changes

• Increased the font weight of the FAQ text to 400
• The links on the during and after section are now white
• Used cursor: pointer; when hovering over the clickable balloons to make sure people can see them

Thanks for the suggestions!

ashayp22 committed:
HackIllinois 2023 Email Header and Footer (#53)

Updated the email header and footer assets for HackIllinois 2023.

srikarmanikonda committed:
Initial commits with UX changes

Fireworks are still goofy, didn't include those changes yet.

srikarmanikonda committed:
Update index.tsx

anandani4136 committed:
:sparkles: Form Navigation Overhaul

Added interactive buttons for the pages in each segment, created all new form pages (page details to be implemented soon) and simplified the code structure for the whack-a-mole navigation

anandani4136 committed:
:memo: Documented Moles File

anandani4136 committed:
:sparkles: Form Question Initial update

anandani4136 committed:
:sparkles: Finishing up form questions

Extended the radiobutton functionality and enhanced some visual elements

anandani4136 committed:
:truck: Form Submission testing

aniketgargya committed:
:bug: just debugging

aniketgargya committed:
:bug: expose app url env variable

anandani4136 committed:
registration testing

ashayp22 committed:
Resolved form validation error

anandani4136 committed:
:iphone: It's Responsive Time

anandani4136 committed:
:lipstick: Refining :white_check_mark: box and select components for :iphone:

anandani4136 committed:
:iphone: :lipstick: Small UI Updates

anandani4136 committed:
:ambulance: Updated Fields for API

aniketgargya committed:
:truck: move registration to /register route

aniketgargya committed:
:truck: rename components to lowercase

aniketgargya committed:
:rotating_light: fix key error warning

aniketgargya committed:
:bug: fix import bug

aniketgargya committed:
:bug: registration validation partially works

anandani4136 committed:
:sparkles: Updates to CSS

anandani4136 committed:
:art: New email headers and footers

aniketgargya committed:
:white_check_mark: fix imports

aniketgargya committed:
:lipstick: reduce max-height rule

anandani4136 committed:
:lipstick: Fixing CSS Bugs part 1

anandani4136 committed:
:lipstick: Fixing CSS Bugs part 2

anandani4136 committed:
:lipstick: Fixing CSS Bugs part 3

- Added custom scrollbars (and fixed scrollbar bugs)
- Made Resume submission optional
- Fixed Finish Screen error with icon going off the screen

anandani4136 committed:
:sparkles: Hello RSVP page (test)

Implemented base RSVP page, ready for initial testing

Notes:
- updated custom.d.ts to allow png types
- link to the discord_username_how_to png is not working yet, will be fixing soon
- api calls yet to be tested

aniketgargya committed:
:sparkles: show event info

anandani4136 committed:
:white_check_mark: fix imports

anandani4136 committed:
:ambulance: Fixed API token for testing

anandani4136 committed:
:bug:  rsvp page fixes

anandani4136 committed:
:bug: rsvp works?

anandani4136 committed:
:zap: smaller images

ashayp22 committed:
Merged with master

anandani4136 committed:
:lipstick: Home Screen Register Button

anandani4136 committed:
:bug: More Bug Fixes

- Functional Question Mark Button
- Updated Formatting for Hero
- Updated Design Consistency

anandani4136 committed:
:art: fixing purdueTransport option

aniketgargya committed:
:rocket: get rid of debugging token

aniketgargya committed:
:building_construction: reroute /rsvp to /register

aniketgargya committed:
Merge pull request #55 from HackIllinois/nextjs-registration

Nextjs registration

anandani4136 committed:
Nextjs registration (#56)

aniketgargya committed:
:lipstick: finish event info section

aniketgargya committed:
Merge branch 'master' into srikareventInfo

aniketgargya committed:
:twisted_rightwards_arrows: fix merge conflicts

aniketgargya committed:
:bug: make event info links clickable

aniketgargya committed:
:page_facing_up: update event info text

aniketgargya committed:
:bug: remove random back slash

aniketgargya committed:
:speech_balloon: remove double "note: "

aniketgargya committed:
:fire: remove old commented code

aniketgargya committed:
:ambulance: add titles to pages

aniketgargya committed:
:lipstick: fix nav spacing

aniketgargya committed:
:bento: add social icons to event info

aniketgargya committed:
:globe_with_meridians: add mobile app urls

aniketgargya committed:
:bento: update favicon

aniketgargya committed:
:alien: remove discord url

aniketgargya committed:
Merge pull request #54 from HackIllinois/eventInfo

Initial commits with UX changes

aniketgargya committed:
Merge branch 'master' into faq-comp

aniketgargya committed:
:white_check_mark: pass build command

aniketgargya committed:
:bug: add faqs to git

aniketgargya committed:
:pencil2: rename faqs to correct name

anandani4136 committed:
:art: Updating the FAQ Text

anandani4136 committed:
:balloon: More FAQ updates
